src/service/resources-impl.js

```diff
@@ -1175,6 +1133,17 @@ export class Resources {
     this.schedulePass(FOUR_FRAME_DELAY_);
   }
 
+  /** @override */
+  measureAndTryScheduleLayout(resource, isPreload, opt_parentPriority) {
+    resource.measure();
+    if (
+      resource.getState() === ResourceState.READY_FOR_LAYOUT &&
+      resource.isDisplayed()
+    ) {
+      this.scheduleLayoutOrPreload_(resource, !isPreload, opt_parentPriority);
```

Comment:
Have you checked every call site to ensure that `isPreload` is the opposite of what it was? Why are we flipping it?

Reply:
Affirm that. The only reason is that we are making this an interface, and it needs to be a bit more readable.